src/sagemaker/local/image.py

```diff
@@ -299,6 +375,55 @@ def retrieve_artifacts(self, compose_data, output_data_config, job_name):
 
         return os.path.join(output_data, "model.tar.gz")
 
+    def write_processing_config_files(
+        self, host, environment, processing_inputs, processing_output_config, processing_job_name
+    ):
+        """Write the config files for the processing containers.
+
+        This method writes the hyperparameters, resources and input data
+        configuration files.
+
+        Returns: None
+
```

Comment:
```suggestion
```

`Returns` section always after `Args`. also, since this is `None`, can remove

Reply:
Done